src/main/java/mat/shared/model/util/MeasureDetailsUtil.java

```diff
@@ -94,4 +99,18 @@ public static String defaultTypeIfBlank(String type) {
         return type == null || type.isEmpty() ? PRE_CQL : type;
     }
 
+    public static Boolean isValidatable(String releaseVersion, boolean draft, boolean draftable, String modelType){
+        BigDecimal version = null;
+        RegExp regExp = RegExp.compile("[0-9]+\\.[0-9]+");
+        MatchResult matcher = regExp.exec(releaseVersion);
+        if (matcher != null) {
+            version = new BigDecimal(matcher.getGroup(0));
+        }
+        if (modelType != null && ((draft && modelType.equals(FHIR)) ||
```

Comment:
you can return the expression value, 
instead of 
    if (expression) { return true; } else {return false}
it would be
    return expression;

Reply:
that would be repetitive of same expression right ?

Reply:
why repetitive? currently you just return true/false. It's not a good practice..

Reply:
Updated